src/App.tsx

```diff
@@ -20,17 +26,152 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    itemWidth: 130,
+    frameSize: 3,
+    step: 3,
+    duration: 0,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  onNext = (step: number) => {
+    const {
+      images, itemWidth, frameSize, duration, infinite,
+    } = this.state;
```

Comment:
```suggestion
  handleNext = (step: number) => {
    const {
      images, 
      itemWidth, 
      frameSize, 
      duration, 
      infinite,
    } = this.state;
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
done